app/controllers/users.js

```diff
@@ -41,3 +52,27 @@ exports.authenticate = (req, res, next) => {
     })
     .catch(reason => next(errors.defaultError(`Database error - ${reason}`)));
 };
+
+exports.list = (req, res, next) => {
+  verifyJwt(req.headers.session_token)
```

Comment:
👍  I don't see it necessary because this is the only expression in the function so it should be returned anyway. If this is for protocol purpose, then it's fine for me, but it seems more like boilerplate to me.